apps/ledger-live-mobile/src/screens/Swap/Form/index.tsx

```diff
@@ -121,7 +125,22 @@ export function SwapForm({
     );
   }, [exchangeRatesState.value, swapTransaction.swap.to.currency]);
 
-  const swapError = swapTransaction.fromAmountError || exchangeRatesState?.error;
+  const isTezosAccountReveal = (swapTransaction: SwapTransactionType): Error | undefined => {
```

Comment:
could you move this fn in LL common ?
as for name, I would rather have "maybeTezosAccountUnrevealedAccount". This name looks more like a test (with bool result)